test/deno/web.test.ts

```diff
@@ -1,17 +1,17 @@
-import { sign, verify } from "../../src/web.ts";
+import { sign, verify } from "../../pkg/dist-web/index.js";
```

Comment:
Shouldn't this be 
```suggestion
import { sign, verify } from "../../pkg/dist-src/web.js";
```

Reply:
![CleanShot 2022-10-04 at 08 08 47@2x](https://user-images.githubusercontent.com/13938334/193707853-fe66d7a2-95bc-4c7e-b375-37c2ec9e8995.png)

If use `dist-src/web.js`, deno's module resolution cannot work..